lib/skin.js

```diff
@@ -27,29 +35,74 @@ module.exports = function () {
     }
   }
 
-  function createSkin(skinId, baseResponsiveSkins) {
-    var skinDir = path.resolve('res/skins', skinId);
-    return new Skin(skinId, skinDir, baseResponsiveSkins);
+  function createSkin(skinId, skinDir) {
+    return new Skin(skinId, skinDir);
```

Comment:
We should add a version check for the method. This should only be supported for 16.4 or greater